### PR TITLE
Add single-story page and images on the feed page

### PR DESCRIPTION
**Pull request by davesteinberg**
This largely completes issues #8 and #63.

It's all still hidden away at `/feed`, but it can be moved to `/` when we deem it ready.

What's still left to do:
- Handle bigger-than-phone screens
- Make the **Share This Story** button functional, or remove it.
- Do the [FSA](https://en.wikipedia.org/wiki/Postal_codes_in_Canada#Forward_sortation_areas) to city mapping?
- Maybe a bit of pixel polishing? The padding doesn't quite match the other pages, but I followed the design. Also, I used `IconButtons` from Chakra, which don't quite match what's in the header.

Also, we should decide if we want to statically render the story pages. I don't think I understand well enough how that would work. Are we going to statically render for every story in the database? How does it know what all those are? Or does it render on demand and then just keep them around?

For now, I'm doing a background fetch when you navigate from the feed to a single story, and server-side rendering when you visit a single story directly. In both cases, you get a request to `stories.get()`, which could be a place to hook in a view count. But we have to decide if that'll be too much load on the database.

### CODE REVIEW

File: pages/story/[id].tsx [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+      <Footer>
+        <Button>Share This Story</Button>
+      </Footer>
+    </Box>
+  )
+}
+
+export async function getServerSideProps({ params }) {
+  try {
+    const prefected = await get(params.id)
```
Comment (silent1mezzo) [inline pages/story/[id].tsx:29]:
```suggestion
    const prefetched = await get(params.id)
```

File: pages/story/[id].tsx [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+    return { props: { prefected } }
```
Comment (silent1mezzo) [inline pages/story/[id].tsx:30]:
```suggestion
    return { props: { prefetched } }
```

**Comment by mwickett** [comment]
@davesteinberg I put up #68  as a test and left some comments. Let me know what you think.

File: pages/story/[id].tsx (hunk 1 above)
Comment (davesteinberg) [reply pages/story/[id].tsx:29]: Good spots, thanks!

Comment (davesteinberg) [reply pages/story/[id].tsx]: Oh, it seems this goes away with Mike's changes. There's always a prefetched story, so it's just called story now.

**Comment by davesteinberg** [comment]
Many thanks @mwickett for the Prisma type fix and the static generation. I collapsed them to one commit so you don't see the Story type being removed and then coming back, and I expanded on the comment on `getStaticPaths()` a bit so that even a dummy like me could understand it.

I've now removed the fetch in the `StoryDetail` component, as well. All seems to be working well. I'm just going to do a few more visual tweaks, then we'll be good to go.

**Review by mwickett — APPROVED** [review]
Looks great!

**Pull request merged**